hosted_control_planes/hcp-import.adoc

```diff
@@ -8,6 +8,17 @@ toc::[]
 
 Hosted clusters are automatically imported into {mce-short} after the hosted control plane becomes available.
 
+include::modules/hcp-import-limitations.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
```

Comment:
See OCP docs guidance on Additional resources section [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections ) and [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies)
```suggestion
[role="_additional-resources_{context}"]
```